src/main/java/com/solidstategroup/bullettrain/BulletTrainClient.java

```diff
@@ -226,6 +226,60 @@ public Trait updateTrait(FeatureUser user, Trait toUpdate) {
         return postUserTraits(user, toUpdate);
     }
 
+    /**
+     * <p>
+     * Create or update a list of user Traits for given user identity.
+     * </p>
+     * <p>
+     * Please note this will override any existing identity with given list.
+     * </p>
+     *
+     * @param user   a user in context
+     * @param traits a list of Trait object to be created or updated
+     * @return a list of added Trait objects
+     */
+    public List<Trait> addTraitsForIdentity(FeatureUser user, List<Trait> traits) {
+        // we are using identities endpoint to create bulk user Trait
+        HttpUrl url = defaultConfig.identitiesURI;
+
+        if (user == null || (user.getIdentifier() == null || user.getIdentifier().length() < 1)) {
+            throw new IllegalArgumentException("Missing user Identifier");
+        }
+
+        // in case we need to delete all the existing Traits
+        // we need to post empty list
+        if (traits == null) {
```

Comment:
This isn't true anymore, nothing will happen to the traits if you post an empty list.

Reply:
fixed